java/src/main/java/com/aliyun/odps/rodps/ROdps.java

```diff
@@ -66,10 +67,15 @@ public class ROdps {
   private String LOGVIEW_HOST;
   private HashMap settings;
 
-
   public ROdps(String projectName, String accessID, String accessKey, String endPoint,
       String dtEndpoint, String logviewHost, String log4j_properties) throws ROdpsException,
       OdpsException {
+     this(projectName, accessID, accessKey, "", endPoint, dtEndpoint, logviewHost, log4j_properties);
+  }
+
+  public ROdps(String projectName, String accessID, String accessKey, String stsToken, String endPoint,
```

Comment:
This new constructor for ststoken is well implemented.
But I find nowhere this constructor is invoked. Maybe you forget add changes to RODPS/R/init.R in your commit?